loader/loader.c

```diff
@@ -2027,7 +2001,7 @@ static VkResult loader_read_layer_json(const struct loader_instance *inst, struc
     if (!strcmp(name, VK_OVERRIDE_LAYER_NAME)) {
         cJSON *expiration;
 
-        if (version.major < 1 && version.minor < 1 && version.patch < 2) {
+        if (version.major == 0 || (version.minor == 1 && version.patch < 2) || version.minor == 0) {
```

Comment:
Why are we looking at patch version here at all?  Any ideas?

Reply:
Apparently 1.1.2 is when the this behavior was added, so its checking that the json version is high enough to support it officially.

Reply:
Ok.  From review I couldn't tell this was a JSON version.